owlbot.py

```diff
@@ -0,0 +1,3 @@
+import synthtool.languages.node as node
```

Comment:
Actually, this is missing a license header

Reply:
Added.